CSS/tablet.css

```diff
@@ -141,3 +168,105 @@
   margin-left: auto;
   margin-right: auto;
 }
+
+/* popup window */
+#popup-window {
+  padding: 3rem 4.5rem;
+  background-color: rgba(193, 199, 208, 0.7);
+  backdrop-filter: blur(9px);
+}
+
+#popup-container {
+  display: flex;
+  flex-direction: column;
+}
+
+#popup-window i.fa-xmark {
+  position: fixed;
+  top: 4.2rem;
+  right: 6rem;
+  font-size: 21px;
+  color: grey;
+}
+
+#popup-window .buttons {
+  display: flex;
+  flex-direction: row;
+  gap: 1rem;
+  align-items: center;
+  justify-content: center;
+}
+
+#popup-window .buttons a {
+  width: fit-content;
+  font-size: 15px;
+}
+
+#popup-window .card {
+  gap: 0;
+}
+
+#popup-window .card img {
+  width: 100%;
+  height: 50%;
+}
+
+#popup-window .card .card-content {
+  width: 100%;
+  gap: 0;
```

Comment:
- [ ] Your modal window does not look like the Figma,kindly resize the image card to fit the figma template .

- [ ] your popup details design
![Screenshot 2023-05-28 015709](https://github.com/Roathy/-Portfolio-/assets/92458236/1157d71e-eeef-4c16-a7ae-d765819778a3)

- [ ] figma design
![Screenshot 2023-05-28 012811](https://github.com/Roathy/-Portfolio-/assets/92458236/445f653c-9171-43a0-9086-1a403baa5a7e)